app/src/main/java/com/cornellappdev/android/eatery/MainListFragment.java

```diff
@@ -393,9 +409,17 @@ public void onClick(int position, ArrayList<EateryBaseModel> list, View viewHold
 
         EateryBaseModel model = list.get(position);
         Intent intent;
+        Bundle bundle = new Bundle();
+        bundle.putString("time", System.currentTimeMillis() + "");
         if (model.isCtEatery()) {
+            mFirebaseAnalytics.logEvent("collegetown_eatery_press", bundle);
             intent = new Intent(getActivity(), CtownMenuActivity.class);
         } else {
+            if (model instanceof DiningHallModel) {
+                mFirebaseAnalytics.logEvent("campus_dining_hall_press", bundle);
+            } else {
+                mFirebaseAnalytics.logEvent("campus_food_court_press", bundle);
```

Comment:
I think food_court could be a little ambiguous. I think in the codebase we call the non-dining hall's cafes so maybe campus_cafe_press is better (I know you suggested this before lol).